### DISCUSSION TITLE
Add the ability to use more precise values in the Accuracy Challenge mod

### DISCUSSION THREAD

**Discussion by Sleepteiner** [ppy/osu#25622] [Ideas]
The Accuracy Challenge mod using the standard accuracy mode (not the maximum achievable accuracy mode) is in most cases that I've found the most efficient way to improve one's accuracy on a map. It is also a lot of fun for me personally to use it for this reason. I have been having a great time using it to improve my scores in the Spotlights and other official playlists. It is honestly the most fun I've had in osu in a long time.

While it is useful and fun, it is also limited in usefulness due to the fact that the player can't set the minimum accuracy to anything other than whole number percentages. If the player gets a 99%+ accuracy score, the mod becomes no longer useful. If the player sets a 98.01% accuracy score, the next minimum accuracy setting above that is 99% which can be a huge jump in difficulty from 98.01%.

My proposal is to allow players to input whatever accuracy they want up to the hundredths place so that this mod could be used to its full potential as a tool for improving scores efficiently. If simply allowing the player to do that makes the mod customization menu look bad or confusing, then the ability to input a precise minimum accuracy could be hidden behind an "extended limits" toggle like in the difficulty adjust mod.

**Comment by huyenden** [comment]
You can adjust the OD as an alternative to the Accuracy Challenge upper limit. 👍

**Reply by Sleepteiner** [reply]
While the combination of the two mods will probably produce interesting results, my proposal is more about being able to use the Accuracy Challenge mod to more efficiently push one's own score on a map to higher levels using the same mods as before rather than actually increasing the difficulty of the map itself. Plus, in the Spotlights, and some other situations, the player can't use the difficulty adjust mod, and for good reason. But, thanks for the idea!

**Reply by huyenden** [reply]
Anyway, good luck on your SS farming journey, I see you have 65% of the SS already

**Comment by quat1024** [comment]
Feeling this on the daily challenge. When the nomod competition gets going, 99% isn't high enough, I want to set it to 99.5% or 99.7%.

Being able to set a maximum for each judgement might be nice too, "restart on >5x100 or >0x50" for example...

**Reply by huyenden** [reply]
Actually, if we compare, I see that they are no different from the perfect mod or the completely increased od to force yourself, anyway, I see that Accuracy Challenge is a measure of how far you can move on the map until you fail by the mod, this will be a more accurate way to use this mod.
